lexicons/tools/ozone/history/defs.json

```diff
@@ -0,0 +1,166 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.history.defs",
+  "defs": {
+    "subjectBasicView": {
+      "type": "object",
+      "required": ["subject", "status", "modAction"],
+      "properties": {
+        "subject": {
+          "type": "string",
+          "format": "uri"
+        },
```

Comment:
I expect that using this field would end-up written `subject.subject`.  Is there any merit in naming it `subject.uri` or `subject.identifier`?